src/utils/ZEnvHelpers.sol

```diff
@@ -212,4 +231,42 @@ library ZEnvHelpers {
         string memory envvar = string.concat(ENV_PREFIX, key);
         return vm.envBool(envvar);
     }
+
+    function __markDirty(State storage s, string memory key) internal {
```

Comment:
__markDirty is only internal vs. private because the state _updating_ logic is still in ZeusScript for some reason

Reply:
maybe we'll move that too in the future